README.md

```diff
@@ -6,7 +6,7 @@
 
 *Copyright (C) 2017 Improbable Worlds Limited. All rights reserved.*
 
-- *Accompanying tutorial*: [https://github.com/spatialos/UnitySDK/blob/master/docs/tutorials/pirates/overview.md](https://spatialos.improbable.io/docs/reference/latest/tutorials/pirates/lesson1)
+- *Accompanying tutorial*: https://github.com/spatialos/UnitySDK/blob/master/docs/tutorials/pirates/overview.md
```

Comment:
If we expose the link, we usually don't include the "https://" to make it a bit neater. Not a biggee, I know! :-)

Reply:
Fixing 👍